src/tests/transaction-flow.ts

```diff
@@ -0,0 +1,291 @@
+import {
```

Comment:
This file aims to test the methods exposed in `Mina.ts` for each implementation of the instance (local blockchain and remote). We switch back and forth and call methods and ensure they behave the same way.

Reply:
great that you added this!